src/components/reusables/Tabs.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+
+export interface TabsProps {
+  items: {
+    [key: string]: JSX.Element;
+  };
+}
+
+export interface TabsState {
+  tab: string;
+}
+
+export default class Tabs extends React.Component<TabsProps, TabsState> {
+  constructor(props: TabsProps) {
+    super(props);
+    this.select = this.select.bind(this);
+    this.makeTabs = this.makeTabs.bind(this);
+    this.state = { tab: Object.keys(this.props.items)[0] };
```

Comment:
Saving the current state as a string and the title of the panel does not feel very stable.

Reply:
What do you think would be better?

Reply:
And id of some kind.